### PR TITLE
Bug 1747260: manifests: move trusted-ca into right spot

### PR DESCRIPTION
**Pull request by mfojtik**
This moves the configmap into right spot (operator namespace). Operator will then copy it.

NOTE: Operator will copy it with the inject label, which means both this operator and networking operator will manage the config map. Since it should always have the right content, I don't see this as a big problem, but it breaks the pattern of "only one controller manage".

### CODE REVIEW

**Comment by mfojtik** [comment]
@deads2k ptal

**Comment by sttts** [comment]
/hold

I don't like this. Why is it not a big problem to have multiple operators modifying the same object? This asks for races.

File: pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00.go [hunk 1]
```diff
@@ -282,28 +277,6 @@ func manageOpenShiftAPIServerImageImportCA_v311_00_to_latest(openshiftConfigClie
 	}
 }
 
-func ensureOpenShiftAPIServerTrustedCA_v311_00_to_latest(client coreclientv1.CoreV1Interface, recorder events.Recorder) error {
```
Comment (sttts) [inline pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00.go]: why is this removed?

**Comment by sttts** [comment]
Before merging anything else, this must be smoke tested in a real cluster.

File: pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00.go (hunk 1 above)
Comment (mfojtik) [reply pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00.go]: not needed, the resource syncer take care of this

**Comment by stlaz** [comment]
/close
in favour of #231 and #232
